.github/workflows/chart_update_on_merge.yml

```diff
@@ -60,7 +60,7 @@ jobs:
       run: |
         set -e
         cd cromwell
-        sbt -Dproject.isSnapshot=false -Dproject.isRelease=false dockerBuildAndPush
```

Comment:
Previously we needed both options to get a `Standard` build, it was a real head-scratcher to figure out. I'm still claiming the interface is unchanged because it's backwards compatible 😄